core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -44,6 +59,18 @@
     @Override
     public void create() {
         pixmap = new Pixmap(WIDTH, HEIGHT, Pixmap.Format.RGBA8888);
+        stage = new Stage();
```

Comment:
für bessere lesbarkeit, bitte sprechende namen verwenden like _pauseStage_ or _gameplayStage_ (In IntelliJ you can refactor names with ALT+F6)

Reply:
Renamed the variable to mainStage. The same stage will be used in the other screens.